### PR TITLE
EWL-3571: adds the course card molecule.

### PR DESCRIPTION
**Pull request by deadarm**
## Ticket(s)

**Jira Ticket**

- [EWL-3571: EC | Design Course Card component - style guide](https://issues.ama-assn.org/browse/EWL-3571)

## Description

Adds the course card molecule to the style guide.

## To Test

- [ ] Navigate to Molecules > Components > Course Card
- [ ] Verify the pattern meets the specifications outlined in https://docs.google.com/document/d/11SpEl1T1uRZ2HPg2g6i-RprZhsPXo_pt2W3xMbCnNZo/edit#heading=h.eou57ct0usj9 and https://issues.ama-assn.org/browse/EWL-3571

## Relevant Screenshots/GIFs
screen shot 2017-06-20 at 11 26 29 am

## Remaining Tasks
N/A

## Additional Notes
N/A

### CODE REVIEW

File: styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.twig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<a class="card card-course" href="#">
+  {% include 'atoms-landscape-3x2' with { 'class': 'card-course_image' } %}
+  <div class="card-course_inner">
+    {% include 'atoms-paragraph' with { 'class': 'card-course_duration', 'content' : '1hr 15min' } %}
+    {% include 'atoms-h2' with { 'class': 'card-course_title', 'content' : 'Association Between Elevated Brain Amyloid and Subsequent Cognitive Decline Among Cognitively Normal Persons' } %}
+    {% include 'atoms-paragraph' with { 'class': 'card-course_credits', 'content' : '3.00 CME Credits'  } %}
```
Comment (rpama) [inline styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.twig]: In the wireframes it looks like the credit hours are bold like the title and start date. Should they be bold here or do we have a reason to prefer they be normal font weight?

File: styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+el: '.card-course'
+title: 'Course Card'
+---
+The Course Card is a variant of the default card style and consists of a required image, course duration, required title, course credits, course date, and optional description, as well a required block level link.  Text fields in this component should have sensible character limits set to enforce a rough maximum height for the cards.
```
Comment (rpama) [inline styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.md]: Can we give recommendations as to what those sensible character limits should be?

**Comment by rpama** [comment]
Oh, one other thing -- didn't see the portrait layout in here, or are we not doing that?

In wireframe:
screen shot 2017-06-21 at 1 25 55 pm

**Comment by deadarm** [comment]
Working on changes :)

File: styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.twig (hunk 1 above)
Comment (deadarm) [inline styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.twig]: Making credit bold.

File: styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.md (hunk 2 above)
Comment (deadarm) [inline styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.md]: Adding 30 char. max to text.

**Comment by deadarm** [comment]
@rpama ready for re-review

**Pull request merged**